stable/thanos/templates/rule-store-api-svc.yaml

```diff
@@ -0,0 +1,37 @@
+{{- with .Values.ruleLayer -}}
+{{- if .enabled -}}
+{{- if .storeApiService.enabled -}}
```

Comment:
Looks like this doesn't exist in the values.yaml provided, when scoped to `.Values.ruleLayer`?  I had to copy in storeApiService from elsewhere.

Reply:
@mikesplain I fixed that, also backported changes from my local repo